GoRogue.Debugger/Routines/RegionRoutine.cs

```diff
@@ -86,27 +100,16 @@ public void GenerateMap()
             foreach (var pos in _map.Positions())
                 _map[pos] = TileState.Wall;
 
-            // Generate some regions of various shapes
-            for (int x = 0; x < _map.Width; x += 50)
+            for (int i = 0; i < 360; i += 45)
             {
-                for (int y = 0; y < _map.Height; y += 50)
-                {
-                    Point here = (x, y);
-                    Region region = GlobalRandom.DefaultRNG.Next(4) switch
-                    {
-                        0 => new Region("arbitrary", (2, 2) + here, (35, 35) + here, (49, 49) + here, (35, 14) + here),
-                        1 => Region.RegularParallelogram("parallelogram", (2, 2) + here, 40, 40, 75),
-                        2 => Region.Rectangle("square", (2, 2) + here, 48, 48),
-                        3 => new Region("triangle", (2, 2) + here, (2, 2) + here, (2, 49) + here, (42, 24) + here),
-                        _ => throw new Exception("Invalid selection for region type.")
-                    };
-                    _originalRegions.Add(region);
-                }
+                var center = (_map.Width / 2, _map.Height / 2);
+                var region = Region.ParallelogramFromTopCorner(center, 15, 15).Rotate(i, center);
+                var overRegion = new RegionWithComponents(region);
+                overRegion.GoRogueComponents.Add(new CenterOfRotation(center));
```

Comment:
If `CenterOfRotation` was removed, this could be:
```suggestion
                overRegion.GoRogueComponents.Add(center, "CenterOfRotation");
```